themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
+
+There are three places in the VuFind configuration that you can change to see your custom theme in action.
+
+The first and most straight-forward is by changing `config.ini > [Site] > theme`. This will change the default theme for everyone who comes to your site and is the setting you should change for production.
+
+The two more development-oriented options are below `theme`: `alternate_themes` and `selectable_themes`. They both take comma-separated lists of themes in the format of `name1:theme1,name2:theme2`. Changing the theme via either of these two options sets a cookie that will keep your theme choice over the course of your session.
+
+Setting up themes under `alternate_themes` allows you to add a `ui=name` parameter to the end of your URL to see that theme. For example, the default configuration of `custom:custom_tmpl` would cause `https://vufind.org/?ui=custom` to show the theme described in the directory `custom_tmpl`.
```

Comment:
Makes sense to me. The only question is how best to do this:

a.) Stand-alone script like install.php
b.) Zend Framework controller action like the language tools, etc.
c.) Phing task

I think of those options I would choose b, since it makes it easiest for you to make use of existing VuFind tools... and Option a is probably the worst (I can't really see any good reason to suggest it, unless we actually build this into the existing install.php somehow, which is problematic for reasons I've previously described). Option c might make the file operations easier to implement, but every other aspect would be harder.

Perhaps you have already reached the same conclusion, or perhaps you have something else in mind. Either way, let me know if I can be of any assistance.